Gemfile

```diff
@@ -23,14 +23,9 @@ gem "faker", "~> 2.14"
 
 # gem "wicked_pdf", "~> 1.4"
 
-gem "daemons"
-gem "delayed_job_active_record"
-gem "delayed_job_web"
-gem "figaro"
 gem "rspec"
 gem "sentry-rails"
 gem "sentry-ruby"
-gem "whenever", require: false
```

Comment:
Could you change this PR to avoid removing `whenever` keeping the usage of the `schedule.rb` file and deleting the `schedule.yml` file?

Reply:
Whenever added again, sidekiq-cron removed because all cron logic will be handle with the whenever gem.